### ISSUE TITLE
Fatal Error: woocommerce-services fatals when Jetpack isn't active

### ISSUE THREAD

**Issue by kraftbj** [Automattic/woocommerce-services#1109]
From 612968-zen:

```
Fatal error: Call to undefined method Jetpack::is_staging_site() in 
/home2/wafg2k0adhre/public_html/wp-content/plugins/woocommerce-services/classes/class-wc-connect-api-client.php 
on line 440
```

We need to check, probably really early, if Jetpack is active or not.

**Comment by v18** [comment]
Follow-up: @kraftbj is asking for more info from the user: plugin version #, what they were trying to do, etc.

**Comment by jeffstieler** [comment]
@kraftbj this was fixed in version 1.7.0 👍
